lua/cfc_time/server/time_tracking.lua

```diff
@@ -105,21 +107,41 @@ function ctime:initPlayer( ply )
     local now = getNow()
     local steamID = ply:SteamID64()
 
+    local function setupPly( totalTime, firstVisit )
+        local sessionTotalTime = totalTime + ( getNow() - now )
+
+        local initialTime = {
+            seconds = 0,
+
+            add = function( this, seconds )
+                this.seconds = this.seconds + seconds
+            end,
+
+            set = function( this, seconds )
+                this.seconds = seconds
+            end
```

Comment:
If we have `add` and `set`, should we not also have a `get` function?

Reply:
okayyy